src/components/Form/Field/Field.scss

```diff
@@ -72,19 +76,21 @@
 
         > div {
           width: 100%;
+          max-width: 64rem;
         }
 
-        &.shrink {
-          width: initial;
+        // &.shrink {
```

Comment:
Will do! Waiting to see if they are going to want them back in there since its in flux.